tests/modules/multivalue_form_element_test/multivalue_form_element_test.info.yml

```diff
@@ -1,5 +1,5 @@
-name: 'Multi-value form element test'
-description: 'Test module for Multi-value form element.'
+name: Multi-value form element test
+description: Test module for Multi-value form element.
 type: module
 package: Testing
 core_version_requirement: ^8 || ^9
```

Comment:
Since we are now putting compatibility to ^8.9 || ^9.1 in composer.json, we can reflect that here.